src/index.js

```diff
@@ -1 +1,23 @@
 import './style.css';
+import postScore from './modules/postscore.js';
+import displayScores from './modules/getscores.js';
+
+// get input from user
+const userInput = document.getElementById('username');
+const scoreInput = document.getElementById('score');
+const submitBtn = document.getElementById('submit');
+
+// add event listener to submit button
+submitBtn.addEventListener('click', () => {
+  postScore(userInput.value, scoreInput.value);
+});
+
+postScore(userInput.value, scoreInput.value);
+
+displayScores();
+
```

Comment:
- [ ] It seems like there's a bug in this code. I noticed that the `postScore()` function is being called outside of the event handler at line `15`, which is causing the user score to be sent to the API multiple times, even though the "Submit" button is only clicked once. This behavior is triggered when the page is refreshed. To fix this issue, I suggest moving the postScore() function inside the "Submit" button's event handler.
- [ ] I'd also suggest moving the `displayScores()` function call inside the event listener for the refresh button